plugins/scaffolder/src/components/TemplatePage/TemplatePage.tsx

```diff
@@ -57,8 +63,16 @@ export const createValidator = (
   rootSchema: JsonObject,
   validators: Record<
     string,
-    undefined | ((value: JsonValue, validation: FieldValidation) => void)
+    | undefined
```

Comment:
This can be changed now right to `CustomFieldValidator`?

Reply:
yepyep i just noticed and have pushed that change